Scripts/validateMavenVersion.ps1

```diff
@@ -43,9 +43,17 @@ $bintrayAPIurl = "https://api.bintray.com/search/packages?name=$packageName"
 $jsonResult = $web_client.DownloadString($bintrayAPIurl) | ConvertFrom-Json
 $bintrayVersion = [version]$jsonResult.latest_version
 
+#If the api calls return empty then this library cannot be compared to the online versions
+#may proceed with the pull request
+if(($mavenVersion -eq $null) -and ($bintrayVersion -eq $null))
```

Comment:
Q: should we use `-or` instead?

Reply:
I wasn't sure here either. Initially I was thinking -or also, however I was think this would only lead to another error if there's a version conflict with either the Maven or Jcenter version and we allow it to proceed. With -and we only allow it to proceed if it has not been published to either repository.